ci/jenkins/jobs/macros.yaml

```diff
@@ -231,7 +231,7 @@
 
           make clean
           docker pull antrea/openvswitch --all-tags
-          docker images | grep 'antrea-ubuntu' | awk '{print $3}' | xargs -r docker rmi || true
+          docker images | grep "${JOB_NAME}" | awk '{print $3}' | xargs -r docker rmi -f || true
```

Comment:
Could you explain what's the final change in this PR and what issue it fixes? I don't understand by the current description.

Reply:
Sorry for not clarifying it.
`docker rmi` cannot delete old images like "antrea-e2e-for-pull-request-99" because `image is referenced in multiple repositories`. Builds triggered by same PR share the same image ID.
To delete these images, `--force` is needed but it may also delete images of other current builds like those triggered by `test-all`.
Since one job isn't allowed to have concurrent builds, it is safe to force delete old images of this kind of job with filter "$JOB_NAME".

Reply:
Thanks for explaining.